Added weekly meeting and old meeting notes (#297)
sunpy/sunpy.org: project/meetings.rst

Meetings
========

Weekly Meetings
***************

> Every Wednedsay at 9am British Time there is a open video meeting that discusses the SunPy project and anything else that people want to cover, like a meeting or just want advice or help.

Reviewer: ```suggestion
Every Wednedsay at 1700 UK time (1600 UTC summer, 1700 UTC winter) there is a open video meeting that discusses the SunPy project and anything else that people want to cover, like a meeting or just want advice or help.
```

`Video Meeting <https://sunpy.org/jitsi>__`

> `We have a Google Calender you can import <https://calendar.google.com/calendar/u/0?cid=ZzljOWVha2c5OGI1Y2JvZ2Q3bTV0YTZoOHNAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ>`__.

Reviewer: We could iframe this. There's also other events in the calendar, so maybe this isn't the best place?

Author: Added.

Meetings
========

Weekly Meetings
***************

> Every Wednedsay at 5pm British Time there is a open video meeting that discusses the SunPy project and anything else that people want to cover, like a meeting or just want advice or help.

Author: See if https://dateful.com/time-zone-converter?t=5pm&tz2=London-UK works for you?

> Every Wednedsay at 5pm British Time there is a open video meeting that discusses the SunPy project and anything else that people want to cover, like a meeting or just want advice or help.

Author: Added.

* `Video Meeting <https://sunpy.org/jitsi>`__

* `Agenda for the meeting <https://demo.hedgedoc.org/GAEnxycXQcCQLrAFN7ie8A?both>`__

> * `Google Calender import <https://calendar.google.com/calendar/u/0?cid=ZzljOWVha2c5OGI1Y2JvZ2Q3bTV0YTZoOHNAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ>`__

Author: ```suggestion
* `Google Calender import <https://calendar.google.com/calendar/u/0?cid=ZzljOWVha2c5OGI1Y2JvZ2Q3bTV0YTZoOHNAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ>`__

.. raw:: html
<iframe src="https://calendar.google.com/calendar/embed?src=g9c9eakg98b5cbogd7m5ta6h8s%40group.calendar.google.com&ctz=America%2FLos_Angeles" style="border: 0" width="800" height="600" frameborder="0" scrolling="no"></iframe>

```

Meetings
========

Weekly Meetings
***************

Every Wednesday at 1700 UK time (1600 UTC summer, 1700 UTC winter, `timezone convertor <https://dateful.com/time-zone-converter?t=5pm&tz2=London-UK works>`__) there is a open video meeting that discusses the SunPy project and anything else that people want to cover, like a meeting or just want advice or help.

* `Video Meeting <https://sunpy.org/jitsi>`__

* `Agenda for the meeting <https://demo.hedgedoc.org/GAEnxycXQcCQLrAFN7ie8A?both>`__

* `Google Calender import <https://calendar.google.com/calendar/u/0?cid=ZzljOWVha2c5OGI1Y2JvZ2Q3bTV0YTZoOHNAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ>`__

.. raw:: html

> <iframe src="https://calendar.google.com/calendar/embed?src=g9c9eakg98b5cbogd7m5ta6h8s%40group.calendar.google.com&ctz=America%2FLos_Angeles" style="border: 0" width="800" height="600" frameborder="0" scrolling="no"></iframe>

Author: ```suggestion
    <iframe src="https://calendar.google.com/calendar/embed?src=g9c9eakg98b5cbogd7m5ta6h8s%40group.calendar.google.com&ctz=America%2FLos_Angeles" style="border: 0" width="800" height="600" frameborder="0" scrolling="no"></iframe>
```

Meetings
========

.. raw:: html

    <iframe id="calendar" src="" width="800" height="600" frameborder="0" scrolling="no"></iframe>
    <script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/jstimezonedetect/1.0.4/jstz.min.js"></script>
    <script type="text/javascript">
    var timezone = encodeURIComponent(jstz.determine().name());
    var iframe_url = `https://calendar.google.com/calendar/embed?src=g9c9eakg98b5cbogd7m5ta6h8s%40group.calendar.google.com&ctz=${timezone}`;
    document.getElementById('calendar').src = iframe_url;
    </script>

Weekly Meetings
***************

> Every Wednesday at 17:00 London time (16:00 UTC summer, 17:00 UTC winter) `timezone convertor <https://dateful.com/time-zone-converter?t=5pm&tz2=London-UK>`__, we hold a community meeting.

Author: ```suggestion
Every Wednesday at 17:00 London time (16:00 UTC summer, 17:00 UTC winter, you can use this `timezone convertor <https://dateful.com/time-zone-converter?t=5pm&tz2=London-UK>`__), we hold a community meeting.
```

> Every Wednesday at 17:00 London time (16:00 UTC summer, 17:00 UTC winter) `timezone convertor <https://dateful.com/time-zone-converter?t=5pm&tz2=London-UK>`__, we hold a community meeting to discuss items related to the project and review open issues and code contributions.

Author: ```suggestion
Every Wednesday at 17:00 London time (16:00 UTC summer, 17:00 UTC winter) `timezone convertor <https://dateful.com/time-zone-converter?t=5pm&tz2=London-UK>`__, we hold a community meeting.
```

The meeting is open to all and we welcome discussion of any SunPy-related topics.
The link to join the meeting as well as a link to the meeting agenda are included below.

* `Link to join the meeting <https://sunpy.org/jitsi>`__

* `Agenda for the meeting <https://demo.hedgedoc.org/GAEnxycXQcCQLrAFN7ie8A?both>`__

> * `Google Calender import <https://calendar.google.com/calendar/u/0?cid=ZzljOWVha2c5OGI1Y2JvZ2Q3bTV0YTZoOHNAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ>`__

Reviewer: ```suggestion
* `Google Calender <https://calendar.google.com/calendar/u/0?cid=ZzljOWVha2c5OGI1Y2JvZ2Q3bTV0YTZoOHNAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ>`__
```

Meetings
========

Weekly Meetings
***************

Every Wednesday at 1700 UK time (1600 UTC summer, 1700 UTC winter, `timezone convertor <https://dateful.com/time-zone-converter?t=5pm&tz2=London-UK>`__) there is a public online meeting that discusses the SunPy project and anything else that people want to cover; like a meeting or if you require support either with Python or ``sunpy``.

> * `Meeting location <https://sunpy.org/jitsi>`__

Reviewer 2: ```suggestion
* `Link to join the meeting<https://sunpy.org/jitsi>`__
```

* `Agenda for the meeting <https://demo.hedgedoc.org/GAEnxycXQcCQLrAFN7ie8A?both>`__

* `Google Calender import <https://calendar.google.com/calendar/u/0?cid=ZzljOWVha2c5OGI1Y2JvZ2Q3bTV0YTZoOHNAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ>`__

> .. raw:: html

Author: Will do

> .. raw:: html

Author: Done.

    <div id="calendar-container"></div>

    <script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/jstimezonedetect/1.0.4/jstz.min.js"></script>
    <script type="text/javascript">
    var timezone = encodeURIComponent(jstz.determine().name());
    var pref = '<iframe src="https://calendar.google.com/calendar/embed?src=g9c9eakg98b5cbogd7m5ta6h8s%40group.calendar.google.com&ctz=';
    var suff = '" style=" border-width:0 " width="800" height="600" frameborder="0" scrolling="no"></iframe>';

>     var iframe_html = pref + timezone + suff;

Author: I guess.

>     document.getElementById('calendar-container').innerHTML = iframe_html;

Author: I copied the code. If it works, we should merge

>     document.getElementById('calendar-container').innerHTML = iframe_html;

Reviewer: It's so ugly even I can tell it's nasty js.

Meetings
========

Weekly Meetings
***************

Every Wednesday at 17:00 GMT+1 / 12:00 EST (16:00 UTC summer, 17:00 UTC winter, `timezone convertor <https://dateful.com/time-zone-converter?t=5pm&tz2=London-UK>`__), we hold a community meeting to discuss items related to the project and review open issues and code contributions.

> The meeting is open to all and we welcome discussion of any SunPy-related topics. The link to join the meeting as well as a link to the meeting agenda are included below.

Author: ```suggestion
The meeting is open to all and we welcome discussion of any SunPy-related topics.
The link to join the meeting as well as a link to the meeting agenda are included below.
```

Meetings
========

Weekly Meetings
***************

Every Wednesday at 17:00 GMT+1 / 12:00 EST (16:00 UTC summer, 17:00 UTC winter, `timezone convertor <https://dateful.com/time-zone-converter?t=5pm&tz2=London-UK>`__), we hold a community meeting to discuss items related to the project and review open issues and code contributions.
The meeting is open to all and we welcome discussion of any SunPy-related topics.
The link to join the meeting as well as a link to the meeting agenda are included below.

> * `Link to join the meeting<https://sunpy.org/jitsi>`__

Author: ```suggestion
* `Link to join the meeting <https://sunpy.org/jitsi>`__
```

>     <div id="calendar-container"></div>
> 
>     <script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/jstimezonedetect/1.0.4/jstz.min.js"></script>
>     <script type="text/javascript">
>     var timezone = encodeURIComponent(jstz.determine().name());
>     var pref = '<iframe src="https://calendar.google.com/calendar/embed?src=g9c9eakg98b5cbogd7m5ta6h8s%40group.calendar.google.com&ctz=';
>     var suff = '" style=" border-width:0 " width="800" height="600" frameborder="0" scrolling="no"></iframe>';
>     var iframe_html = pref + timezone + suff;
>     document.getElementById('calendar-container').innerHTML = iframe_html;
>     </script>

Reviewer: ```suggestion
    <iframe id="calendar" src="" width="800" height="600" frameborder="0" scrolling="no"></iframe>
    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jstimezonedetect/1.0.4/jstz.min.js"></script>
    <script type="text/javascript">
    var timezone = encodeURIComponent(jstz.determine().name());
    var iframe_url = `https://calendar.google.com/calendar/embed?src=g9c9eakg98b5cbogd7m5ta6h8s%40group.calendar.google.com&ctz=${timezone}`;
    document.getElementById('calendar').src = iframe_url;
    </script>
```

Weekly Meetings
***************

> Every Wednesday at 17:00 GMT+1 / 12:00 EST (16:00 UTC summer, 17:00 UTC winter, `timezone convertor <https://dateful.com/time-zone-converter?t=5pm&tz2=London-UK>`__), we hold a community meeting to discuss items related to the project and review open issues and code contributions.

Reviewer: This edit isn't correct, the time actually changes during the year, the only "accurate" way to say it is that it happens at 1700 London time.

```suggestion
Every Wednesday at 17:00 London time (16:00 UTC summer, 17:00 UTC winter, `timezone convertor <https://dateful.com/time-zone-converter?t=5pm&tz2=London-UK>`__), we hold a community meeting to discuss items related to the project and review open issues and code contributions.
```

> Every Wednesday at 17:00 GMT+1 / 12:00 EST (16:00 UTC summer, 17:00 UTC winter, `timezone convertor <https://dateful.com/time-zone-converter?t=5pm&tz2=London-UK>`__), we hold a community meeting to discuss items related to the project and review open issues and code contributions.

Reviewer: ```suggestion
Every Wednesday at 17:00 GMT+1 / 12:00 EST (16:00 UTC summer, 17:00 UTC winter, `timezone convertor <https://dateful.com/time-zone-converter?t=5pm&tz2=London-UK>`__), we hold a community meeting to discuss items related to the project such as reviewing open issues and code contributions.
```

> The meeting is open to all and we welcome discussion of any SunPy-related topics.

Reviewer: We just said a much more limited description of the meeting topic above and now we are saying we are open to talking about anything.

Author: Not anymore.

* `Clayton Hotel Burlington Road <https://bookings.claytonhotelburlingtonroad.com>`_, Leeson Street Upper, Dublin, D04 A318
* `Mespil Hotel <https://www.mespilhotel.com>`_, 50-60 Mespil Rd, Dublin 4, D04 E7N2
* `Pembroke Townhouse <https://www.pembroketownhouse.ie>`_, 88 Pembroke Rd, Dublin, D04 TF72

> Previous SunPy Coordination Meeting
> ***********************************

Reviewer: ```suggestion
Previous SunPy Coordination Meetings
************************************
```
